src/NzbDrone.Core/Notifications/DiscordNotifier/DiscordNotifier.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Linq;
+using FluentValidation.Results;
+using NzbDrone.Common.Extensions;
+
+namespace NzbDrone.Core.Notifications.DiscordNotifier
+{
+    public class DiscordNotifier : NotificationBase<DiscordNotifierSettings>
+    {
+        private readonly IDiscordNotifierProxy _proxy;
+
+        public DiscordNotifier(IDiscordNotifierProxy proxy)
+        {
+            _proxy = proxy;
+        }
+
+        public override string Link => "https://discordnotifier.com";
+        public override string Name => "Discord Notifier";
```

Comment:
Can we flip the name to `DiscordNotifier.com` it seems to cause less confusion with users who can't figure out which one to use.